Assets/HoloToolkit/Input/Scripts/InputSources/DictationInputManager.cs

```diff
@@ -103,11 +103,12 @@ protected override void OnDestroy()
         /// <summary>
         /// Turns on the dictation recognizer and begins recording audio from the default microphone.
         /// </summary>
+        /// <param name="listener">GameObject listening for the dictation input.</param>
         /// <param name="initialSilenceTimeout">The time length in seconds before dictation recognizer session ends due to lack of audio input in case there was no audio heard in the current session.</param>
         /// <param name="autoSilenceTimeout">The time length in seconds before dictation recognizer session ends due to lack of audio input.</param>
         /// <param name="recordingTime">Length in seconds for the manager to listen.</param>
         /// <returns></returns>
-        public static IEnumerator StartRecording(float initialSilenceTimeout = 5f, float autoSilenceTimeout = 20f, int recordingTime = 10)
+        public static IEnumerator StartRecording(GameObject listener, float initialSilenceTimeout = 5f, float autoSilenceTimeout = 20f, int recordingTime = 10)
```

Comment:
What if I'm using a global listener and I don't want/need to pass a specific GameObject in? Can we give `listener` a default value (probably `null`), like we've done with the others? That'd also prevent this from being a breaking change.

Reply:
Good question. 

I don't think it's a good idea to make the object you're using to start the dictation a global listener.
But if you did, you'd have a script on the object that listens for the click handler:

```
OnInputClicked(EventData eventData)
{
    StartCoroutine(DictationManager.Instance.StartRecording(gameObject);
{
```

Reply:
Ahh, I didn't mean the starting object being a global listener, but the one being sent the dictation results. If I want a button to start recording, but I want the results to be sent to a global listener (or two), my button would be forced to pass in some GameObject anyway.

Reply:
I don't think I follow.

Reply:
Okay, I'll make the listener optional.